Google.Api.Generator.Utils/Roslyn/RoslynBuilder.cs

```diff
@@ -315,9 +315,11 @@ static void AppendString(string s, List<InterpolatedStringContentSyntax> list)
         public static SwitchFunc Switch(PropertyDeclarationSyntax expr) => Switch(IdentifierName(expr.Identifier));
 
         public static CastExpressionSyntax Cast(TypeSyntax type, ExpressionSyntax expr) => CastExpression(type, expr);
-
+        
```

Comment:
Nit: remove this whitespace-only change?

Reply:
👍